daemon/src/main/java/org/sonarlint/daemon/services/ConnectedSonarLintImpl.java

```diff
@@ -0,0 +1,177 @@
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.daemon.services;
+
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonarlint.daemon.services.AbstractSonarLint.DefaultClientInputFile;
+import org.sonarlint.daemon.services.AbstractSonarLint.ProxyIssueListener;
+import org.sonarlint.daemon.services.AbstractSonarLint.ProxyLogOutput;
+import org.sonarsource.sonarlint.core.ConnectedSonarLintEngineImpl;
+import org.sonarsource.sonarlint.core.client.api.common.analysis.ClientInputFile;
+import org.sonarsource.sonarlint.core.client.api.connected.ConnectedAnalysisConfiguration;
+import org.sonarsource.sonarlint.core.client.api.connected.ConnectedGlobalConfiguration;
+import org.sonarsource.sonarlint.core.client.api.connected.ConnectedGlobalConfiguration.Builder;
+import org.sonarsource.sonarlint.core.client.api.connected.ConnectedSonarLintEngine;
+import org.sonarsource.sonarlint.core.client.api.connected.ConnectedSonarLintEngine.State;
+import org.sonarsource.sonarlint.core.client.api.connected.ServerConfiguration;
+import org.sonarsource.sonarlint.daemon.proto.ConnectedSonarLintGrpc.ConnectedSonarLint;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.ConnectedAnalysisReq;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.ConnectedConfiguration;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.LogEvent;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.ModuleUpdateReq;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.ServerConfig;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.StorageStatus;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Void;
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.StorageStatus.Status;
+
+import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.InputFile;
+
+import java.nio.charset.Charset;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.List;
+
+public class ConnectedSonarLintImpl implements ConnectedSonarLint {
+  private static final Logger logger = LoggerFactory.getLogger(ConnectedSonarLintImpl.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)